tree_points_topic: "/tree_points"
bev_occ_topic: "/bev_occ"
output_dir: "/mysda/w/w/lio_ws/output/debug_stage1"
save_mode: "trigger"
save_rate_hz: 1.0
max_files: 0

grid_res: 0.05
grid_x_min: 0.0
grid_x_max: 12.0
grid_y_min: -4.0
grid_y_max: 4.0
point_size: 1
save_occ_bev: true
use_occ_grid_dims: true
